source/server/admin/stats_handler.cc

```diff
@@ -104,6 +104,14 @@ Http::Code StatsHandler::handlerStats(absl::string_view url,
     }
   }
 
+  for (const Stats::CounterGroupSharedPtr& counter : server_.stats().counterGroups()) {
+    if (shouldShowMetric(*counter, used_only, regex)) {
+      for (size_t i = 0; i < counter->maxEntries(); ++i) {
+        all_stats.emplace(absl::StrCat(counter->name(), counter->nameSuffix(i)), counter->value(i));
+      }
```

Comment:
just a quick note that this will explode memory during operation. There's an ongoing Issue (https://github.com/envoyproxy/envoy/issues/16139) and some experimental PRs to address, which would affect this new loop: